### PR TITLE
[#158653881] Create Json Web Tokens and Add more Tests

### PR DESCRIPTION
**Pull request by RachaelNantale**
`Business/User Value:_ As a User, I should have access to only the specified pages

Acceptance Criteria
WHEN User navigates to the ride offers
THEN User should be able to authenticate the pages making sure a logged in person has the right access`

### CODE REVIEW

File: UML/ride-my-way.xml [hunk 1]
```diff
@@ -1 +0,0 @@
...CjzNsAMIIPj3h0Kvg4zTpf0bw6RWKXgUffatvBJ8OI9yr4OPUSXqZ4FP3OGUG/+XQ6FXtGTWJgVraMXJg74C0FWO6AaTpFlKrMQaQ3gFxW04u3bww0hecizb211gLIpgu2f5jZsgzcEodQFxI/UHFIS+BTKGMQ/+DXOkERZ+ggIX+lw0Jkx25w6Ck1QC3vWgQXzYUNG/H9IKI7biHG/wzVQMILn6Nm4LuqilaOu5oyDmmcKf+jAvnP2s83erdwmcY43/IIV4JtNHDjB6W18Mct6iHjTT9J1PTqIs730mTICbnKJi1Fs94QzxqtLFzTFxp4vg+tLFJ042NEUN6haJXhWwyaoBiDj0moeGiPy56FckmTUvlBAnwQrCFb/jomY/WEls3gLjHAIm20NMFCw0m/WOiH1dPLjboBJO6xVVGbD9ziN2W95WdPHLUvU6RicDjGAmzvvI9pIBJrzNF09sYH1NsAHkPgAytlnNEJ4BoyaQkwH1O3tNVQDASnJHg8hLceFyR4K6t/E/N5DipY1sbzwu3WYJygZSXppA3SXG1VHSS8Yzq8l65qFPjLsfFUdWl9qHasHE5NuoUucux0SS4hLtgibl8ubPVy/gjg8UFsagV4i7HRVVhUS/74k4kr/sMDhfEoU5wuxgO+pMcRnDrMsTjlnePXQhus6Z3/Mj308WtMgUYNaUnOuqWp10uATQ9W3go9DA1hLwHQmYXnCL0BHBsRsitLGRbXFxSaA8KSwqHRSiSpkrL+GoVnaPrAZMlv1WH1QmSrZcEquYeJA2ZXjQs6UX2NP1slW4kWd+ojss8XWlqZJ9sKul1pSnwNjrkqiliy2FNPdEu0nWPCM2RVsM1F8z8MlYYExu2ZiGi+bWs+IWIv+Uog34mpX++GYSjq0f1/+63WD36hsWi9eG0S4v5nKn1Oi6cEheONWvJxWuiXn0AUE4vxz5WoXNpMhf5Ezl7qWTpJi05r0rnpd9M4lWfbVRnH6RfQzzxr/UJ0o3a75X9tpSgx+2oeARbrSL85rhUX4zGbxteExZ97/u9xOVbw9J6LmvxsaYzZTj72rJnuRx3ZV1DDlKGB8wJXLx8sTS3zpf8nG7y2nhSSkdOqYm2ec2dVF61dJXVar5S8QNltYY4vzartR0+Nd9L+IFS0nGnnsmnUMw+1ZtwnX0P2fn4Pw==</diagram></mxfile>
```
Comment (malmike) [inline UML/ride-my-way.xml:1]: Why are you deleting the UML

File: app/__init__.py [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
 from .api.views.user_views import app_bps
+from .api.views.requests_views import app_register
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token, get_jwt_identity)
 
 
 def create_app(config_module):
     app = Flask(__name__)
 
+    app.config['JWT_SECRET_KEY'] = 'sup3rsecr3t'
```
Comment (malmike) [inline app/__init__.py:13]: It would be advisable to get the secret key from the environment settings

File: dbHandler.py [hunk 3]
```diff
@@ -64,10 +178,10 @@ def close(self):
...
-    ride.fetch_all('RequestTable')
-    ride.fetch_one('RideTable')
+    # ride.fetch_all('RequestTable')
+    # ride.fetch_one('RideTable')
 
     # ride.create_record("UserTable", "(id, username,email,password, phone)",
     #                    ('1yyqqqy', 'rachael', 'rachaelexample.com', '', '1111111111'))
-    # ride.create_record("RideTable", "(id, driver,pickup_point,destination, time, done)",
-    #                    ('1yeq', 'rachael', 'Kamwokya', 'busia', '7pm', False))
+    # ride.create_record("RequestTable", "(request_id, ride_id, passenger,pickup_point,destination, time, status)",
```
Comment (malmike) [inline dbHandler.py:186]: Remove all commented out code

File: UML/ride-my-way.xml (hunk 1 above)
Comment (RachaelNantale) [reply UML/ride-my-way.xml:1]: In order to separate challenge 2 from 3

**Pull request merged**